ipn/state_enum_encoding.go

```diff
@@ -0,0 +1,56 @@
+// Code generated by go-enum-encoding DO NOT EDIT
+package ipn
```

Comment:
You attached this to the "package" line so it's now package doc.

The generator should be fixed.

Reply:
good point. fixing this. able to repro.

<img width="613" alt="image" src="https://github.com/user-attachments/assets/1db0ce40-c3f7-4ada-bbdb-8372ed8eb985">

Reply:
fixed.

<img width="955" alt="image" src="https://github.com/user-attachments/assets/f55908d3-5176-47dd-a706-79108bfcacdd">